### DISCUSSION TITLE
Port issue

### DISCUSSION THREAD

**Discussion by s3kler** [pwndoc/pwndoc#480] [General]
Hi, I am running into an issue installing Pwndoc. It fails to install because port 8443 is already running with a different container and I was wondering if it is possible to change the port so it doesn't interfere with the other container. I tried to change it in the docker file but it didn't work. Thanks!

**Comment by yeln4ts** [comment]
Try changing also in:
`.docker/nginx.conf`
![image](https://github.com/pwndoc/pwndoc/assets/4255028/8aee9f1c-1adf-4713-b92d-4763a8fdc6ac)
and in `quasar.conf.js`
![image](https://github.com/pwndoc/pwndoc/assets/4255028/f5d9d82b-dc03-4859-83d6-18a1ba8d57b1)

**Comment by s3kler** [comment]
Thanks for that, it worked.
